CMakeLists.txt

```diff
@@ -107,6 +107,14 @@ endif()
 # as defined above.
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/Binaries)
 
+if (MSVC)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
```

Comment:
Should those be indented to match the rest of the file wrt to control structures?